scripts/lua/header_authenticated.lp

```diff
@@ -68,7 +68,7 @@ mg.include('header.lp','r')
                         <ul class="dropdown-menu">
                             <!-- User image -->
                             <li class="user-header">
-                                <img class="logo-img" src="<?=pihole.webhome()?>img/logo.svg" alt="Pi-hole Logo" style="border: 0" width="90" height="90">
+                                <img class="logo-img" src="<?=pihole.webhome()?>img/logo.svg" alt="Pi-hole Logo" width="90" height="90">
```

Comment:
this is redundant because `img` has `border: none` set in CSS. Generally, we should avoid inline CSS when possible.